modules/study/src/main/StudyApi.scala

```diff
@@ -415,7 +415,18 @@ final class StudyApi(
     sequenceStudyWithChapter(studyId, setTag.chapterId):
       case Study.WithChapter(study, chapter) =>
         Contribute(who.u, study):
-          doSetTags(study, chapter, PgnTags(chapter.tags + setTag.tag), who)
+          if (setTag.tag.name == Tag.Event) {
+            val data: ChapterMaker.EditData = ChapterMaker.EditData(
+              chapter.id,
+              StudyChapterName(setTag.tag.value),
+              ChapterMaker.Orientation.Auto,
+              ChapterMaker.Mode.Normal,
+              chapter.description.getOrElse("-")
+            )
+            editChapter(studyId, data)(who)
+          } else {
+            doSetTags(study, chapter, PgnTags(chapter.tags + setTag.tag), who)
+          }
```

Comment:
I don't think the PGN event should change the chapter name.

Example: https://lichess.org/study/nptBfbK6

All chapters have the same Event tag, but different chapter names.